images/build/Dockerfiles/gitHubActions.Dockerfile

```diff
@@ -75,6 +76,13 @@ RUN if [ "${DEBIAN_FLAVOR}" = "bookworm" ]; then \
             libssl1.1 \
             libyaml-dev \
             libxml2 \
+            # Adding lxml depended packages to avoid build failures
+            # https://lxml.de/installation.html#requirements
+            libxml2-dev \
+            libxslt-dev \
+            python3-dev \
+            python3-setuptools \
+            python3-wheel \
```

Comment:
Do we know what binaries these packages bring in? Specifically, are they going to conflict with any of the Python binaries (such as `python`, `python3`, `pip`, etc.) that we may be installing via dynamic installation?

Reply:
These shouldn't cause conflicts. `python3-dev` package has the most dependencies installed with it, but I believe we've already installed them in the past. It's just the old way to install it isn't getting picked up during pip install (pip install used to throw issues). Normally, user can install/upgrade these from pip. From my understanding, this is just an alternative way to install without tempering with build script generator.